src/ui-config/marketsConfig.tsx

```diff
@@ -237,6 +238,23 @@ export const marketsData: {
       UI_INCENTIVE_DATA_PROVIDER: markets.AaveV3Goerli.UI_INCENTIVE_DATA_PROVIDER,
     },
   },
+  [CustomMarket.proto_sepolia_v3]: {
```

Comment:
Maybe we should add this in the list before goerli show it shows up first and as the default v3 testnet?

Reply:
good call- updated in `8c664765204adcd9339b259ae89eb1932da6db6b`